### PR TITLE
adding chapters

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- adding chapters

### CODE REVIEW

File: website/forms/features/geolocation.md [hunk 1]
```diff
@@ -3,6 +3,56 @@ id: geolocation
...
+
+![Geolocation modal screen](/img/forms/geolocation-modal.webp)
+
+### Rule preview
+
+This toggle is here to help you preview what you have configured so far. It will show you all the forms shown to the user based on the geolocation rules.
+
+## Cloudflare
+
+If you have Cloudflare turned on you your project please make sure that you read the [Cloudflare documentation](cloudflare) to make sure that the geolocation feature works as expected.
```
Comment (piqusy) [inline website/forms/features/geolocation.md:58]:
```suggestion
If you have Cloudflare turned on in your project please make sure that you read the [Cloudflare documentation](cloudflare) to make sure that the geolocation feature works as expected.
```

**Review by piqusy — CHANGES_REQUESTED** [review]
Looking good 👏🏼 

Just one wording suggestion

File: website/forms/features/geolocation.md [hunk 2]
```diff
@@ -3,6 +3,56 @@ id: geolocation
...
+
+## Where do you get the country list from?
+
+We use DataHub to list countries for the dropdown menu to select the country of usage.
+
+[All releases are listed here](https://datahub.io/core/country-list) and our implementation is located on [this link](https://github.com/infinum/eightshift-forms/blob/develop/data/country/manifest.json).
+
+With the default countries list, we have made a few groups:
+* Europe
+* Europian Union
```
Comment (iobrado) [inline website/forms/features/geolocation.md:26]:
```suggestion
* European Union
```

**Review by iobrado — CHANGES_REQUESTED** [review]
One minor change 🙂

**Review by iobrado — APPROVED** [review]

**Review by piqusy — APPROVED** [review]

**Pull request merged**